nix/build/hacks.nix

```diff
@@ -57,13 +57,13 @@ rec {
 
   # FIXME opam-nix needs to do version resolution
   ezjsonm = osuper.ezjsonm.versions."1.2.0";
-  hacl-star-raw = osuper.hacl-star-raw.overrideAttrs (o: rec {
+  hacl-star-raw = osuper.hacl-star-raw.versions."0.3.2".overrideAttrs (o: rec {
     preConfigure = "patchShebangs raw/configure";
     sourceRoot = ".";
     buildInputs = o.buildInputs ++ [ self.which ];
     propagatedBuildInputs = buildInputs;
   });
-  hacl-star = osuper.hacl-star.overrideAttrs (_: rec {
+  hacl-star = osuper.hacl-star.versions."0.3.2".overrideAttrs (_: rec {
```

Comment:
Uh these changes in cfa4a24c9e522b27f5ca869fb241d2cbd3e88478, revert part of the changes in the commit just before (665d95a134cdce848828a1ba4a72caeb465051b8).

If both v10 and v9 now need this, can't these commits be merged?

(if you do so, please mention it in the commit message tho)

Reply:
hmm, yeah, looks like it's safe to squash them